templates/example-environments/rdo-cloud-env.yaml

```diff
@@ -8,3 +8,29 @@ parameters:
   undercloud_image: CentOS-7-x86_64-GenericCloud-1703
   undercloud_flavor: m1.large
   toci_jobtype: multinode-1ctlr-featureset004
+  undercloud_user_data: |
+    #!/bin/bash
+    rpm -q git || yum -y install git
+    git clone https://git.openstack.org/openstack-infra/tripleo-ci
+    export HOME=/root
+    tripleo-ci/scripts/tripleo.sh --repo-setup
+    yum -y install python-heat-agent*
+    rm -f /etc/yum.repos.d/delorean*
+    rm -rf tripleo-ci
+    systemctl daemon-reload
+    systemctl enable os-collect-config
+    reboot
```

Comment:
yes, perhaps it is, but I'm not sure how to tackle it with custom scripts, like those I wanted for my [case](https://github.com/bogdando/traas/blob/69719d31532db833c28aa2980c6b51384f42c6f4/templates/example-environments/rdo-cloud-warp-env.yaml#L11-L21).